### DISCUSSION TITLE
[Self Hosted] How to create multiple users for dashboard with different permissions

### DISCUSSION THREAD

**Discussion by Tejaswgupta** [supabase/supabase#27306] [Questions]
I've self hosted Supabase using Docker and I want to share the access with some of my team members. However, I want to keep the access to non-admin , so no destructive functions can be executed. I can invite users on the managed Supabase, how can I do the same on the self hosted one?

Thanks for your response.

**Comment by Varaddin1** [comment]
Did you get the answer for this issue, running into it now

**Comment by M7MMAD-OMAR** [comment]
Self-hosted Studio has a single basic-auth login, so everyone who can open it has full rights. The usual workaround is to keep Studio off the internet (SSH tunnel or VPN) and give read-only people a Postgres role with `SELECT` grants instead of Studio access.

Disclosure: I built an open source layer on top of self-hosted Supabase, Sbarbase, that adds owner, admin and viewer roles with invitations. Viewers can see the project but can't open Studio, and Studio itself opens per environment through a short-lived ticket instead of a shared password.

https://github.com/M7MMAD-OMAR/sbarbase
